src/pages/Home/Home.styles.ts

```diff
@@ -0,0 +1,26 @@
+const main = {
+	width: '100%',
+	height: '80vh',
+	marginTop: '30px',
+	display: 'flex',
+	justifyContent: 'center',
+	gap: '30px',
+};
+const container = {
+	display: 'flex',
+	flexDirection: 'column',
+	alignItems: 'center',
+	gap: '80px',
+};
+const card = {
+	display: 'flex',
+	flexDirection: 'column',
+	justifyContent: 'center',
+	alignItems: 'center',
+	padding: '20px',
+	borderRadius: '4px',
+	boxShadow: '2px 2px 2px 1px rgba(0, 0, 0, 0.2)',
+	backgroundColor: '#fff',
+	maxWidth: '600px',
+};
+export { main, container, card };
```

Comment:
Import this file as a default module.

```suggestion
 const styles =  { main, container, card };
 export default styles;
```